Scripts/Tracking/Follow/Modifier/SourceFollowActiveTarget.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace VRTK.Core.Tracking.Follow.Modifier
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Forces the transformations of the source <see cref="Transform"/> to match those of the target <see cref="Transform"/> by applying the given <see cref="FollowModifier"/>.
+    /// </summary>
+    public class SourceFollowActiveTarget : FollowModifier
+    {
+        /// <summary>
+        /// The <see cref="FollowModifier"/> to apply to the source <see cref="Transform"/> that is following the active target <see cref="Transform"/>.
+        /// </summary>
+        [Tooltip("The FollowModifier to apply to the source Transform that is following the active target Transform.")]
+        public FollowModifier appliedModifier;
+
+        /// <summary>
+        /// Updates the source <see cref="Transform"/> position using the first active target <see cref="Transform"/>.
+        /// </summary>
+        /// <param name="source">The source <see cref="Transform"/> to modify.</param>
+        /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>
+        public override void UpdatePosition(Transform source, Transform target)
+        {
+            if (appliedModifier != null && isActiveAndEnabled)
+            {
+                appliedModifier.UpdatePosition(source, target);
+                CachedSource = appliedModifier.CachedSource;
+                CachedTarget = appliedModifier.CachedTarget;
+            }

```

Comment:
This logic is repeated in each subclass of `FollowModifier`. I wonder if we should change the methods `FollowModifier.Update*` to be non-abstract and implement this, then call a `protected` method that actually implements the "real" `Update*`?

In the superclass `FollowModifier`:
```c#
public virtual void UpdatePosition(Transform source, Transform target)
{
    if (appliedModifier != null && isActiveAndEnabled)
    {
        DoUpdatePosition(source, target);
        CachedSource = appliedModifier.CachedSource;
        CachedTarget = appliedModifier.CachedTarget;
    }
}

protected abstract void UpdatePosition(Transform source, Transform target);
```

In the subclass `SourceFollowActiveTarget`:
```c#
protected override void UpdatePosition(Transform source, Transform target)
{
    appliedModifier.UpdatePosition(source, target);
}
```

In the subclass `TargetsFollowSource`:
```c#
protected override void UpdatePosition(Transform source, Transform target)
{
    appliedModifier.UpdatePosition(target, source);
}
```

I always resort to the `DoMethodName` notation for this. (Plus the same for the other `Update*` methods.)